pom.xml

```diff
@@ -23,7 +23,7 @@
     <dependency>
       <groupId>org.apache.hive</groupId>
       <artifactId>hive-serde</artifactId>
-      <version>2.3.4</version>
+      <version>2.3.6</version>
```

Comment:
Let's create a `hive.version` property up above and reference it here and on line 78 so we only have one place to update it in future.